src/enclave/enclave_init.c

```diff
@@ -244,7 +251,7 @@ int __libc_init_enclave(int argc, char** argv)
     __init_libc(envp, argv[0]);
     __init_tls();
 
-    size_t futex_wake_spins = sgxlkl_enclave->shared_memory.vvar ? 1 : 500;
+    size_t futex_wake_spins = 1;
```

Comment:
This should be a documented macro, in case we want to change this.

Reply:
I have no idea what it does, so I'll leave the documentation to others. Do you want an actual macro, or did you intend to say attested config setting?

Reply:
Something like:
```
// Configures after how many scheduler cycles futexes are woken up
#define DEFAULT_FUTEX_WAKE_SPINS 1
```

This should also move to the lthread scheduler.

Reply:
What is "this"? The macro, the variable, the code block, the call to lthread_sched_global_init and where is "the scheduler"? Do you mean that this is not the right place to call `lthread_sched_global_init`?

Reply:
I don't think that `futex_wake_spins` should be a parameter to `lthread_sched_global_init`. Instead, we can use the constant directly inside `lthread.c`.

Reply:
Got it, done!